examples/storm-elasticsearch-examples/src/main/java/org/apache/storm/elasticsearch/common/EsTestUtil.java

```diff
@@ -56,7 +57,7 @@ public static Tuple generateTestTuple(final String source,
         GeneralTopologyContext topologyContext = new GeneralTopologyContext(
                 builder.createTopology(),
                 new Config(),
-                new HashMap<>(),
+                new CustomIndexArray<String>(0,1),
```

Comment:
Nit: java should be able to figure out that it is a `String` from the required type for GeneralTopologyContext

Reply:
It is mostly for readability .. when reading this code its unclear what the deduced type will be without further checking the parameter types for the constructor. Happy to change if you prefer the other way.